src/vm/excep.cpp

```diff
@@ -6549,6 +6552,8 @@ DWORD GetGcMarkerExceptionCode(LPVOID ip)
 #if defined(HAVE_GCCOVER)
     WRAPPER_NO_CONTRACT;
 
+    // IsGcCoverageInterrupt(ip) requires "gccover.h"
```

Comment:
Seems like you could leave the declaration of `IsGcCoverageInterrupt` in jitinterface.h, and then you wouldn't need to change this file. Or jitinterface.h.

Reply:
No that doesn't work fort he DAC